template/module/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © <YEAR(s)> <AUTHOR(s)>
+
+# Instructions: https://www.odoo.com/documentation/8.0/reference/module.html
```

Comment:
Is this useful enough? I worry that people will leave it around then using the template.